cnf-certification-test/lifecycle/scaling/scaling.go

```diff
@@ -27,96 +27,221 @@ import (
 
 	v1app "k8s.io/api/apps/v1"
 	v1autoscaling "k8s.io/api/autoscaling/v1"
-	v1 "k8s.io/client-go/kubernetes/typed/apps/v1"
 
 	v1machinery "k8s.io/apimachinery/pkg/apis/meta/v1"
+	hps "k8s.io/client-go/kubernetes/typed/autoscaling/v1"
 	retry "k8s.io/client-go/util/retry"
+)
 
-	hps "k8s.io/client-go/kubernetes/typed/autoscaling/v1"
+const (
+	deployment  = "deployment"
+	statefulset = "statefulset"
 )
 
-func TestScaleDeployment(deployment *v1app.Deployment, timeout time.Duration) bool {
+func isDeploymentInstanceReady(deployment *v1app.Deployment) bool {
```

Comment:
This function is already available in "github.com/test-network-function/cnf-certification-test/cnf-certification-test/lifecycle/podsets". You can jut reuse it